database/sql/V010__create_app_data.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE app_data (
+    app_data_hash bytea PRIMARY KEY,
+    version bytea NOT NULL,
+    app_code bytea NOT NULL,
+    referrer bytea NOT NULL,
```

Comment:
Both are option and should be nullable

Reply:
I would also add indexes for `app_code`, `referrer`